app/views/casework/crime_applications/_client_details.html.erb

```diff
@@ -29,7 +29,7 @@
       <%= label_text(:date_of_birth) %>
     </dt>
     <dd class="govuk-summary-list__value">
-      <%= l applicant.date_of_birth, format: :compact %>
+      <%= l applicant.date_of_birth, format: :dob %>
```

Comment:
I seem to remember caseworkers want DoB it in the same format they paste into MAAT. Does this change that?